### PR TITLE
Add Appium

### PR DESCRIPTION
**Pull request by SrinivasanTarget**
**[https://github.com/SrinivasanTarget/awesome-appium]**

**[A curated list of delightful Appium resources. Appium is an open source, cross-platform test automation tool for native, hybrid and mobile web apps, tested on simulators (iOS, FirefoxOS), emulators (Android), and real devices (iOS, Android, Windows, FirefoxOS).]**

# ALL THE BELOW CHECKBOXES SHOULD BE DONE AND CHECKED AND THE ABOVE PLACEHOLDERS SHOULD BE FILLED IN BEFORE SUBMITTING A PULL REQUEST!

- [x] I have read and understood the [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/contributing.md) and the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/master/create-list.md).
- [x] This pull request has a descriptive title. *(For example: `Add Node.js`)*
- The list I added
	- [x] **has been around for at least 20 days,**
	- [x] is a non-generated Markdown file in a GitHub repo,
	- [x] is not a duplicate,
	- [x] only has awesome items with descriptions *(Awesome lists are curations of the best, not of everything)*,
	- [x] contains the [awesome badge](https://github.com/sindresorhus/awesome/blob/master/awesome.md#awesome-badge) on the right side of the list heading,
	- [x] has a Table of Contents section named `Contents` as the first section,
	- [x] has an [appropriate license](https://github.com/sindresorhus/awesome/blob/master/awesome.md#choose-an-appropriate-license) *(Meaning something like CC0, NOT a code licence like MIT, BSD, Apache, etc)*,
	- [x] has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/master/awesome.md#include-contribution-guidelines),
	- [x] has been checked for inconsistent formatting, spelling, and grammar *(This includes starting each link description with an uppercase character and ending it with a period `.`, and consistent naming)*,
	- [x] has been added at the bottom of the appropriate category.

https://github.com/SrinivasanTarget/awesome-appium

### CODE REVIEW

**Comment by SrinivasanTarget** [comment]
@sindresorhus Any update on this?

File: readme.md [hunk 1]
```diff
@@ -379,6 +379,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [Open Companies](https://github.com/opencompany/awesome-open-company)
 - [REST](https://github.com/marmelab/awesome-rest)
 - [Selenium](https://github.com/christian-bromann/awesome-selenium)
+- [Appium](https://github.com/SrinivasanTarget/awesome-appium)
```
Comment (sindresorhus) [inline readme.md]: Can you add a very short description? See how it's done elsewhere.

**Comment by sindresorhus** [comment]
Generally looks good :)

> This includes starting each link description with an **uppercase character and ending it with a period .**, and consistent naming)

Not followed

-

- There's no reason to use a table for the `Clients` section. Better to just have a list and linkify the entries.
- A few entries are missing description.

**Comment by SrinivasanTarget** [comment]
@sindresorhus Addressed all the review comments.Please check.

**Pull request merged**